From 13b9806ff811197a13c34e2778d5151c040220f6 Mon Sep 17 00:00:00 2001
From: Rob Clark <rob@ti.com>
Date: Sat, 2 Oct 2010 17:49:49 -0500
Subject: [PATCH 27/32] v4l2sink: account for min-queued-bufs

Need to include min-queued-bufs in the calculation of required number of
buffers.
---
 sys/v4l2/gstv4l2sink.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sys/v4l2/gstv4l2sink.c b/sys/v4l2/gstv4l2sink.c
index 21e3548..ffccba6 100644
--- a/sys/v4l2/gstv4l2sink.c
+++ b/sys/v4l2/gstv4l2sink.c
@@ -810,7 +810,7 @@ gst_v4l2sink_set_caps (GstBaseSink * bsink, GstCaps * caps)
      * on the pipeline?
      */
     if (min_buffers != -1) {
-      min_buffers += 4;
+      min_buffers += 3 + v4l2sink->min_queued_bufs;
     }
 
     if (min_buffers > v4l2sink->num_buffers) {
-- 
1.7.1

